.github/workflows/test.yml

```diff
@@ -10,24 +10,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # Earliest and latest version of each major version
-        php-versions: ['7.2', '7.4', '8.0', '8.2']
+        # Earliest and latest version of each major version, that is not EoL
+        php-versions: ['8.1', '8.3']
         prefer: ['prefer-stable', 'prefer-lowest']
-        symfony-versions: ['^3.4', '^4', '^5', '^6']
+        symfony-versions: ['^5.4', '^6', '^7']
         include:
-            - symfony-versions: '^3.4'
-              symfony-bundle-test-version: '^1.8'
+            - symfony-versions: '^7'
+              symfony-bundle-test-version: '2.0.x-dev'
+              symfony7-required-yaml-version: '^6 || ^7'
```

Comment:
symfony-bundle-test-version can be omitted, if version 3 works with every configuration - will test tomorrow.
symfony7-required-yaml-version is required for `composer --prefer-dist prefer-lowest update`, because it would install symfony/yaml 5.4 resulting in the following error, executing phpunit.
`PHP Fatal error:  Declaration of Symfony\Component\Yaml\Command\LintCommand::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output) must be compatible with Symfony\Component\Console\Command\Command::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output): int in [...]versioning-bundle/vendor/symfony/yaml/Command/LintCommand.php on line 91